tools/devops/device-tests/scripts/GitHub.psm1

```diff
@@ -311,6 +311,7 @@ function New-GitHubSummaryComment {
         $TestSummaryPath,
 
         [String]
+        [AllowEmptyString()]
```

Comment:
This will be a merge issues with https://github.com/xamarin/xamarin-macios/pull/8977 but was needed to test the full pipeline.